AdditionalTask2/AdditionalTask2/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Globalization;
+
+namespace AdditionalTask2;
+
+public class PositivOrNegative
+{
+    public static void Main()
+    {
+        IFormatProvider formatter = new NumberFormatInfo { NumberDecimalSeparator = "." };
```

Comment:
For what we need this here?

Reply:
Find this there https://metanit.com/sharp/tutorial/20.4.php
Because in different language can be different symvol for floating point number